lib/engine/game/g_18_co/game.rb

```diff
@@ -1495,7 +1495,9 @@ def emergency_issuable_cash(corporation)
         end
 
         def emergency_issuable_bundles(entity)
-          issuable_shares(entity)
+          eligible, remaining = issuable_shares(entity)
```

Comment:
what's going on here, do you have a bad rebase?

Reply:
I think that's the case, I did a rebase last week to keep this branch from falling too far behind but I had to resolve some conflicts, I may have messed up some stuff

Reply:
A `git rebase master; git push --force` fixed it